metapath/src/Helpers.c

```diff
@@ -2222,17 +2222,15 @@ static LRESULT CALLBACK DefaultFileDlgHookProc(HWND hWnd, UINT uMsg, WPARAM wPar
 			BOOL bPathChanged = FALSE;
 			HWND hCmbPath = GetDlgItem(hWnd, cmb13); // cmb13: dlgs.h
 			int n = GetWindowText(hCmbPath, szPath, MAX_PATH);
-			if (n > 0 && PathFileExists(szPath)) {
-				for (int i = 0; i < n; i++) {
-					if (szPath[i] == TEXT('/')) {
-						szPath[i] = TEXT('\\');
-						bPathChanged |= TRUE;
-					}
-				}
-				if (bPathChanged) {
-					SetWindowText(hCmbPath, szPath);
+			for (int i = 0; i < n; i++) {

```

Comment:
This loop can be replaced with PathFixBackslashes() (can be changed to returns BOOL with input path is fixed).

Reply:
Ooh, I thought there should be a function like this. Optimized.